setup.py

```diff
@@ -9,6 +9,6 @@
     license='MIT',
     install_requires=[],
     dependency_links=['git+ssh://git@github.com/sourcebots/robotd.git#egg=robotd'],
```

Comment:
Not _entirely_ related to this PR, but seeing as `robotd` isnt an explicit requirement of using `robot-api`, we should probably remove it from `dependency_links` too

Reply:
Good point, and I would consider it relevant to this PR. You don't need to have `robotd` installed in order to use `robot-api`; you just need something that's compatible with `robotd`'s API.